core/src/main/scala/cromwell/core/actor/BatchActor.scala

```diff
@@ -72,14 +74,28 @@ abstract class BatchActor[C](val flushRate: FiniteDuration,
 
   def commandToData(snd: ActorRef): PartialFunction[Any, C]
 
+  // If work has arrived 'recently' - ie within this duration - ignore any periodic 'ScheduledProcessAction' messages
+  val recentArrivalThreshold: Option[FiniteDuration] = None
```

Comment:
This val can never change and stores no information. Does that mean line 89 is a `None foreach { ... }`? What does that even do?